### ISSUE TITLE
Drop down tabbing

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#10235]
**What**
- Viewer keyboard focus needs to be improved
- Download menu in viewer inaccessible to keyboard users

**Where**
Viewer whilst using a keyboard

Related to #10236?

**TODO**

- [x] Make download menu accessible to keyboard users

**Background**

From the Hassell Inclusion accessibility audit August 2023.

Dovetail: https://proficient-wallaby-dpwd.dovetailapp.com/insights/Viewer-keyboard-focus-needs-to-be-improved-394ZEhb0ZwJ3ufkLAtHImF

https://proficient-wallaby-dpwd.dovetailapp.com/insights/Download-menu-in-viewer-inaccessible-to-keyboard-users-3PYiUvLhqmhYM3jNIpb2E

Both are WCAG fails

Miro board: https://miro.com/app/board/uXjVMjsL5YE=/
